From 2f357ab39fe1c1fd0e059538f84e37dbf09527d7 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Wed, 4 Jul 2018 13:47:07 -0400
Subject: [PATCH 4856/5725] Revert "drm/amd/display: Fix indentation in dcn10
 resource constructor"

That change was a merge gone bad.

This reverts commit cb1d7eacb58f7d1b7d0e57b26dc02d45eada4a3c.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dcn10/dcn10_resource.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dcn10/dcn10_resource.c b/drivers/gpu/drm/amd/display/dc/dcn10/dcn10_resource.c
index 0a313dc..c92a156 100644
--- a/drivers/gpu/drm/amd/display/dc/dcn10/dcn10_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/dcn10/dcn10_resource.c
@@ -1068,8 +1068,7 @@ static bool construct(
 
 	ctx->dc_bios->regs = &bios_regs;
 
-		pool->base.res_cap = &res_cap;
-
+	pool->base.res_cap = &res_cap;
 	pool->base.funcs = &dcn10_res_pool_funcs;
 
 	/*
-- 
2.7.4

